Config.cmake/mergestaticlibs.cmake

```diff
@@ -90,6 +90,11 @@ EXECUTE_PROCESS(COMMAND ls .
         add_custom_command(TARGET ${outlib} POST_BUILD
             COMMAND ${CMAKE_COMMAND} -E echo "Running: ${CMAKE_RANLIB} $<TARGET_FILE:${outlib}>"
             COMMAND ${CMAKE_RANLIB} $<TARGET_FILE:${outlib}>)
+    else()
+        # only works with static third-party libs
+        foreach(libtarget ${libs})
+            target_sources(${outlib} PUBLIC $<TARGET_OBJECTS:${libtarget}>)
+        endforeach()
     endif()
```

Comment:
Can you explain this a bit more, and how it solves the Mingw issue in #192?

1. Reading the doc it's not clear to me how it's valid that target_sources(...) can accept object files. To me it looks like it expects .c and .h files, and that one should use target_link_libraries(...) with the object files?
2. How can we use $<TARGET_OBJECTS:...> if libtarget is not specified as add_library(... OBJECT)? At least I don't see that we specify it as OBJECT.

Reply:
it seems #192 build failed because it did not have the ls command to list object files

I tried several things including creating an intermediate OBJECT library but it did not work but curiously target_sources worked, so seems I have to dig on the cmake side